lib/class-wp-rest-menu-items-controller.php

```diff
@@ -251,6 +275,13 @@ protected function prepare_item_for_database( $request ) {
 			}
 		}
 
+		$taxonomy = get_taxonomy( 'nav_menu' );
+		$base     = ! empty( $taxonomy->rest_base ) ? $taxonomy->rest_base : $taxonomy->name;
+		// If menus submitted, then check if 1 menu is given, if more than 1 is given, error out.
```

Comment:
I think this comment needs to be adjusted now.

Reply:
Fixed in https://github.com/WP-API/menus-endpoints/pull/31/commits/cbc17732475ae99893696091debfa3d59dc41e18